app/src/interfaces/_system/system-filter/input-component.vue

```diff
@@ -39,9 +39,8 @@ const displayValue = computed(() => {
 	return props.value;
 });
 
-const width = computed(() => {
-	return (props.value?.toString().length || 2) + 1 + 'ch';
-});
+const inputLength = ref<number>();
```

Comment:
Seeing this input length is based on the HTML input value, which in turn is based on the `value` prop, we gotta make sure we reset this number when the value prop is updated from outside in with a watcher